config.md

```diff
@@ -4,9 +4,17 @@ Aardvark-dns will read configuration files from a given directory.
 
 Inside this directory there should be at least one config file. The name of the file equals the network name.
 
+### First line
 The first line in the config must contain a comma separated list of listening ips for this network, usually the bridge ips.
 At least one ip must be given.
+**Note**: Optional second coloumn allows users to specifify DNS servers at network level (seperated by comman), all the containers
+will inherit following nameservers instead of using host's resolver.
 
```

Comment:
you show a nice example below of what this looks like in the container.  is an example applicable here too?

Reply:
I included all the examples in the last just like we do it in podman , adding another example here look redundant in docs. But I can add it if its needed.